### DISCUSSION TITLE
有没有哪位老师申请过教师教育资源包？Unfortunately, we were unable to verify your academic affiliation based on the information

### DISCUSSION THREAD

**Discussion by WJC-POG** [community/community#51452] [GitHub Education]
Has anyone ever been rejected when applying for teachers' benefits?
I've applied almost 20 times but keep getting rejections. I've tried uploading my work documents, course schedule, and employment contract. I am no longer sure what other official documents I need to provide to prove that I am a university teacher.
有人在申请教师资源时被拒绝过吗？
我已经申请了近20次，但一直被拒绝。我试着上传了我的工作文件、课程表和就业合同。我已经不确定我还需要提供哪些正式文件来证明我是一名大学教师。

**Comment by WJC-POG** [comment]
@Mntwana here is my discussion.
I have already filled in the name and bio, and then I reapplied, but I still can't get through.

**Reply by Teddy-van-Jerry** [reply]
Sad here that I cannot apply for the student pack too... GitHub's verification seems too rigid so I cannot get through after several failed attempts (especially when the system cannot read Chinese documents...).

**Comment by Ntshangase** [comment]
It seems you are doing everything correct, maybe try using the computers camera to capture the University ID Card (if you weren't already),

**Reply by Ntshangase** [reply]
OHhhh!!!!!, Try  creating a repository just to 'complete' your GitHub profile.

**Reply by WJC-POG** [reply]
Unfortunately, the solution you describe does not seem to be working.

**Comment by WJC-POG** [comment]
@samus-aran 
I've applied 22 times but keep getting rejections. I've tried uploading my work documents, course schedule, and employment contract. I am no longer sure what other official documents I need to provide to prove that I am a university teacher.

Here is a list of all the errors I got so far:
Unfortunately, we weren't able to approve your educational discount request for [@WJC-POG](https://github.com/WJC-POG).


What happened?
The item you uploaded is insufficient to demonstrate your current academic status.
Please see suggestions.
Suggested proof to expedite your approval:

You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.


Have you completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with all your relevant information, such as your full name as it appears in your image and a short bio?


If you meant to apply as a student, please cancel your application and reapply [here](https://education.github.com/discount_requests/student_application).


Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.

**Reply by newlulu** [reply]
I have tried to apply it 6 times. All reject.

**Comment by samus-aran** [comment]
Hi @WJC-POG 

Let's see how we can assist ⭐ 

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

If your original document is not in English then we advise you to take a photo of the **original** next to an English translation. This also include your profile information.

Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Let us know how you get on 👍

**Reply by WJC-POG** [reply]
Thank you for your reply, but I'm applying for teacher's benefits, and my latest attempt to apply was to upload a photo of my work and school certificates together, but it still didn't go through.
Here are the relevant tips:
![image](https://user-images.githubusercontent.com/53817132/230543385-27bc27e9-0720-4b71-9d8c-4dd9a68025e9.png)
![image](https://user-images.githubusercontent.com/53817132/230543499-5c25472e-0221-496a-ac47-afa0ed064afc.png)

**Reply by xdouworld** [reply]
Have you passed now? I am a student, I also encountered this mistake. If you pass, I hope you can share your experience in this discussion. Thank you very much

**Reply by AliAmmarNaseem** [reply]
Same here!
